examples/v1beta1/taskruns/hub-resolver.yaml

```diff
@@ -17,11 +17,39 @@ spec:
   taskRef:
     resolver: hub
     params:
-      - name: catalog # optional
-        value: Tekton
+      - name: type  #optional
+        value: artifact
       - name: kind
         value: task
       - name: name
         value: git-clone
       - name: version
         value: "0.6"
+---
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  generateName: hub-resolver-semver-
+spec:
+  workspaces:
+    - name: output
+      emptyDir: {}
+  podTemplate:
+    securityContext:
+      fsGroup: 65532
+  params:
+    - name: url
+      value: https://github.com/tektoncd/pipeline.git
+    - name: revision
+      value: main
+  taskRef:
+    resolver: hub
+    params:
+      - name: type  #optional
+        value: artifact    
+      - name: kind
```

Comment:
`kind` is optional too, right?

can we have an example with the required fields only?